app/ports/migrations/0004_add-fields-to-stats.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.1.7 on 2019-07-27 13:03
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ports', '0003_add-field-cxx_stdlib'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='build_arch',
+            field=models.CharField(default='', max_length=20),
+        ),
+        migrations.AddField(
+            model_name='submission',
+            name='platform',
+            field=models.CharField(default='', max_length=20),
```

Comment:
I'm not 100% sure, but we could actually use `darwin` as default here :)

Similar question as above: could the migration automatically set `darwin`; this time simply to all existing entries?

Reply:
> I'm not 100% sure, but we could actually use `darwin` as default here :)
> 
> Similar question as above: could the migration automatically set `darwin`; this time simply to all existing entries?

If we had set `default=darwin`, then this migration would automatically set `darwin` in existing entries.

I would add `darwin` as default, thanks.